### ISSUE TITLE
Maven: Could not find fbjni-0.5.0-headers.jar

### ISSUE THREAD

**Issue by tomasgarba** [facebookincubator/fbjni#87]
## Issue description

Missing [fbjni-0.5.0-headers.jar](https://repo.maven.apache.org/maven2/com/facebook/fbjni/fbjni/0.5.0/fbjni-0.5.0-headers.jar) on Maven.

Previous version 0.4.0 contains all headers files but 0.5.0. doesn't.

**Comment by ishaan1995** [comment]
We are also facing same issue with react-native-vision-camera in our RN app
```
* What went wrong:
Execution failed for task ':react-native-vision-camera:extractAARHeaders'.
> Could not resolve all files for configuration ':react-native-vision-camera:extractHeaders'.
   > Could not find fbjni-0.5.0-headers.jar (com.facebook.fbjni:fbjni:0.5.0).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/facebook/fbjni/fbjni/0.5.0/fbjni-0.5.0-headers.jar
```

**Comment by gguidotti** [comment]
Having the same issue with react-native-vision-camera package.
I saw the commit addressing missing headers in 0.5.1 version but I'm still seeing above package looking inside 0.5.0 folder instead of the 0.5.1 one, not sure why

**Comment by tomasgarba** [comment]
We are using `extractHeaders("com.facebook.fbjni:fbjni:+:headers")` + convention to grab latest version but pipeline is still using 0.5.0 version. I guess it will be solved automatically in few minutes/hours while new version 0.5.1 will be properly available.

**Comment by rodrigob-infocorp** [comment]
Same issue here, we are trying to workaround it by modifying the build.gradle of the library project to use 0.5.1 instead. 

I hope the 0.5.0 version gets the headers files soon.

**Comment by ishaan1995** [comment]
> Same issue here, we are trying to workaround it by modifying the build.gradle of the library project to use 0.5.1 instead.
> 
> I hope the 0.5.0 version gets the headers files soon.
@rodrigob-infocorp what modification are you applying to the build.gradle? Can you share if possible?

**Comment by tomasgarba** [comment]
It's simple just change to `extractHeaders("com.facebook.fbjni:fbjni:0.4.0:headers")` but unfortunately other libraries such as mmkv using plus convention. I could patch this but I hope 0.5.1. will start working soon.

**Comment by jzamituy** [comment]
Same issue, we are using mmkv also, do you if this is going to be fixed today? We want to avoid CI to fail.
Thanks guys.

**Comment by rodrigob-infocorp** [comment]
> > Same issue here, we are trying to workaround it by modifying the build.gradle of the library project to use 0.5.1 instead.
> > I hope the 0.5.0 version gets the headers files soon.
> > @rodrigob-infocorp what modification are you applying to the build.gradle? Can you share if possible?

Didn't work, there are some extra references inside this third party library that breaks when using the headers 0.5.1 :(

**Comment by cortinico** [comment]
Issue has already been discussed and resolved here: https://github.com/facebookincubator/fbjni/issues/85

tl;dr: Version 0.5.1 was published with headers and all the required files.
